fed/_private/serialization_utils.py

```diff
@@ -63,7 +63,8 @@ def find_class(self, module, name):
 def _apply_loads_function_with_whitelist():
     global _pickle_whitelist
 
-    _pickle_whitelist = fed_config.get_cluster_config().serializing_allowed_list
+    _pickle_whitelist = fed_config.get_job_config() \
+        .cross_silo_comm_config.serializing_allowed_list
```

Comment:
Should this be job config?
@zhouaihui @fengsp CC

Reply:
Now that we separate `ray.init` from `fed.init`,  there's no way to reach the cluster-level information, since each `fed.init` starts and only starts a job session.

Unless there's a global actor (or service job) that can break the job isolation and filter each job's tasks' invalid param type.

Reply:
If we don't have the cluster config, why we not just use `config`? The question that we should answer before it getting finalized is whether we need the cluster config in the future at high level.

Reply:
Let's discuss in #156